agreement_sale_condition_delivery_planning/__manifest__.py

```diff
@@ -8,10 +8,11 @@
     "development_status": "Alpha",
     "category": "Sale",
     "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/contract",
     "maintainers": ["gurneyalex"],
     "license": "AGPL-3",
     "application": False,
-    "installable": True,
+    "installable": False,
```

Comment:
You are disabling a module. That can't be done without a strong reason.

Reply:
Sorry for not explaining...

The module was added a few days ago coming from v12: https://github.com/OCA/contract/commit/1602e240aee8337788924836f45f467f9f620865#diff-691bfd59e98c623738e91124f0f2c1bb00e4f5219a2cecb0380c2cac7f5541dc

It seems not to be migrated, as the version still indicates v12, so I think it is better to just ignore it until it is migrated... That seems to be what is done in some other situations like these...

Reply:
Seeing the commit, I can have even been an error, where the new module shouldn't be uploaded... Perhaps @gurneyalex can give some insight...

Reply:
It seems he had some dirty files in his local repository. Better to revert such commit directly. Please do it in a separate PR.

Reply:
Ok, done in https://github.com/OCA/contract/pull/716 :+1: